wiki/Tournaments/GBC/GBC_2022_Autumn/en.md

```diff
@@ -173,6 +174,59 @@ The GB Cup 2022 Autumn is run by the Team GB and various community members.
 
 ## Match results
 
+### Reugular stage summary
+
+| #[^rs-summary-seed] | Teams | abbr.[^rs-summary-abbr] | MR[^rs-summary-mr] | BR[^rs-summary-br] | BD[^rs-summary-bd] | BU[^rs-summary-bu] | Round 1 | Round 2 | Round 3 | Round 4 |
+| :-: | --: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: |
+| 1 | DAMWONGaming | DWG | 1-0 | 6-0 | 6 | -1 | 6-0 不玩SV | 猫猫面馆 |
+| 2 | 奖励一把大天空 | 奖励天空 | 1-0 | 6-1 | 5 | -1 | 6-1 俄亥俄 | ζ(s) |
+| 3 | 又一个倒霉蛋 | 倒霉蛋 | 1-0 | 6-1 | 5 | -1 | 6-1 绿皮龙 | 行李箱 |
+| 4 | 铁戈金马 | 铁戈金马 | 1-0 | 6-2 | 4 | -1 | 6-2 队名难取 | 不为所动 |
+| 5 | 均衡教派 | 均衡教派 | 1-0 | 6-2 | 4 | -1 | 6-2 开摆 | 屙屎精灵 |
+| 6 | 经典米托镂空鱼香鸡肉丸 | 肉丸子 | 1-0 | 6-2 | 4 | -1 | 6-2 素鲍小馆 | 近月少女 |
+| 7 | 近月少女的礼仪 | 近月少女 | 1-0 | 6-2 | 4 | -1 | 6-2 BMS | 肉丸子 |
+| 8 | 我屙屎精灵我交泥如活打屙 | 屙屎精灵 | 1-0 | 6-3 | 3 | -1 | 6-3 凑不齐人 | 均衡教派 |
+| 9 | 为所动做为所欲为的所欲为 | 不为所动 | 1-0 | 6-3 | 3 | -1 | 6-3 炒树莓 | 铁戈金马 |
+| 10 | ban位先ban行李箱 | 行李箱 | 1-0 | 6-3 | 3 | -1 | 6-3 你说的队 | 倒霉蛋 |
+| 11 | 黎曼猜想 | ζ(s) | 1-0 | 6-4 | 2 | -1 | 6-4 石头门 | 奖励天空 |
+| 12 | 猫猫面馆 | 猫猫面馆 | 1-0 | 6-5 | 1 | -1 | 6-5 传统音游 | DWG |
+| 13 | 都什么年代还在玩传统音游 | 传统音游 | 0-1 | 5-6 | -1 | 1 | 5-6 猫猫面馆 | 不玩SV |
+| 14 | EIPSYKONGROO | 石头门 | 0-1 | 4-6 | -2 | 1 | 4-6 ζ(s) | 俄亥俄 |
+| 15 | 你说的队 | 你说的队 | 0-1 | 3-6 | -3 | 1 | 3-6 行李箱 | 绿皮龙 |
+| 16 | 番茄炒树莓 | 炒树莓 | 0-1 | 3-6 | -3 | 1 | 3-6 不为所动 | 队名难取 |
+| 17 | 上课军训补课反正凑不齐人 | 凑不齐人 | 0-1 | 3-6 | -3 | 1 | 3-6 屙屎精灵 | 开摆 |
+| 18 | BMS精灵 | BMS | 0-1 | 2-6 | -4 | 1 | 2-6 近月少女 | 素鲍小馆 |
+| 19 | 素鲍小馆丸子面条铺 | 素鲍小馆 | 0-1 | 2-6 | -4 | 1 | 2-6 肉丸子 | BMS |
+| 20 | 要我一直摆烂一直摆烂吗 | 开摆 | 0-1 | 2-6 | -4 | 1 | 2-6 均衡教派 | 凑不齐人 |
+| 21 | 队名太多不知道用哪个 | 队名难取 | 0-1 | 2-6 | -4 | 1 | 2-6 铁戈金马 | 炒树莓 |
+| 22 | 无所谓绿皮龙会出手的 | 绿皮龙 | 0-1 | 1-6 | -5 | 1 | 1-6 倒霉蛋 | 你说的队 |
+| 23 | 俄亥俄州立大学躁狂播放器 |  | 俄亥俄 | 0-1 | 1-6 | -5 | 1 | 1-6 奖励天空 | 石头门 |
+| 24 | SV狗都不玩 | 不玩SV | 0-1 | 0-6 | -6 | 1 | 0-6 DWG | 传统音游 |
+
+### Regular stage week 1
+
+Saturday, 29 October 2022:
+
+| Team A |  |  | Team B | Match link |
+| --: | --: | :-- | :-- | :-: |
```

Comment:
please follow the [template](https://github.com/ppy/osu-wiki/blob/master/wiki/Tournaments/TEMPLATE.md?plain=1#L129)

```suggestion
| --: | :-: | :-: | :-- | :-- |
```

Reply:
might template update... so sry just copy last year's format.